pkg/master/master_test.go

```diff
@@ -112,6 +115,61 @@ func setUp(t *testing.T) (*etcdtesting.EtcdTestServer, Config, *assert.Assertion
 	return server, *config, assert.New(t)
 }
 
+// TestLegacyRestStorageStrategies ensures that all Storage objects which are using the generic registry Store have
+// their various strategies properly wired up. This surfaced as a bug where strategies defined Export functions, but
+// they were never used outside of unit tests because the export strategies were not assigned inside the Store.
+func TestLegacyRestStorageStrategies(t *testing.T) {
+	_, _, masterCfg, _ := newMaster(t)
+	storageProvider := corerest.LegacyRESTStorageProvider{
+		StorageFactory:       masterCfg.StorageFactory,
+		ProxyTransport:       masterCfg.ProxyTransport,
+		KubeletClientConfig:  masterCfg.KubeletClientConfig,
+		EventTTL:             masterCfg.EventTTL,
+		ServiceIPRange:       masterCfg.ServiceIPRange,
+		ServiceNodePortRange: masterCfg.ServiceNodePortRange,
+		LoopbackClientConfig: masterCfg.GenericConfig.LoopbackClientConfig,
+	}
```

Comment:
Unfortunately had to test at the master package level. Ideally this would have been with the LegacyRESTStorageProvider, however this requires substantial plumbing to instantiate so we could get at the internal storage map, all of which existed here in the master package. This package is also the location where it is used.

I can re-attempt this if desired however it will mean a much larger patch to copy or refactor code, as it's a lot of plumbing and there's additionally a circular dependency at play between the two packages.